app/packages/operators/src/state.ts

```diff
@@ -123,6 +130,7 @@ const useExecutionContext = (operatorName, hooks = {}) => {
         extended,
         filters,
         selectedSamples,
+        selectedLabels,
       },
       hooks
     );
```

Comment:
(Github doesn't allow me to comment on the actual line below)

Shouldn't the line below (the second argument of `useMemo`) also be adapted to include `selectedLabels` as well?

i.e. change it from:
```
}, [params, datasetName, view, extended, filters, selectedSamples, hooks]);
```

to
```
}, [params, datasetName, view, extended, filters, selectedSamples, selectedLabels, hooks]);
```

Reply:
I’ll make this change. Curious that the example plugin still seems to work. Passing the arrays here may not function as expected.

Reply:
Yeah, I also wonder why. Probably one of the other dependencies gets updated too.